doc/release-notes-26471.md

```diff
@@ -0,0 +1,16 @@
+Updated settings
+----------------
+
+- Previously running with the `-blocksonly` option effectively disabled the
+  mempool but continued to allocate 300MB of cache for it, which would be
+  shared with the UTXO cache. This might lead to nodes running with this
+  option using more resources than anticipated, especially during IBD.
+
+  In this release running with `-blocksonly` will only allocate 5MB to the
+  mempool by default, permitting the use-case of relaying transactions from
+  whitelisted peers, but avoiding unexpected resource usage. It reduces the
+  available `dbcache` size by 295MB, impacting performance especially during
```

Comment:
Perhaps need to add here that this is only true if `-maxmempool` is not set? We say it in other places, but better to not assume everyone reads everything, so seems like a safe option.